src/stories/LineVis.stories.tsx

```diff
@@ -44,6 +44,16 @@ Domain.args = {
   domain: [-100, 100],
 };
 
+export const Abscissas = Template.bind({});
+
+Abscissas.args = {
+  dataArray,
+  domain,
+  abscissaParams: {
+    value: Array.from({ length: dataArray.size }, (_, i) => -10 + 0.5 * i),
+  },
+};
+
```

Comment:
Moved up from `Display` folder, as I think it's as important a feature as the error bars and auxiliary arrays.